students/synowiec_krzysztof/lesson_04_unit_testing/test_the_length_of_the_segment.py

```diff
@@ -0,0 +1,33 @@
+import pytest
+from math import sqrt
+
+
+# Function under test
+def distance(x1, y1, x2, y2):
+    return sqrt((x1 - x2) ** 2 + (y1 - y2) ** 2)
+
+
+def test_distance_none_parameters():
+    with pytest.raises(TypeError):
+        distance(None, None, None, None)
+
+
+def test_distance_string_parameters():
+    # Test that it crashes with ValueError when called on 'aoeu' - why?
```

Comment:
There will be ValueError when casting to float/ing would be inside function and it would raise an exception e.g.
`float('aoeu')`